1st-gen/packages/menu/src/MenuItem.ts

```diff
@@ -719,6 +719,13 @@ export class MenuItem extends LikeAnchor(
 
     updateAriaSelected(): void {
         const role = this.getAttribute('role');
+        if (role === 'menuitem') {
+            if (this.selected) {
+                this.setAttribute('aria-current', 'true');
+            } else {
+                this.removeAttribute('aria-current');
+            }
+        }
```

Comment:
I'd prefer to keep `aria-current` as a separate concern from selection.

`role` is determined by either the `role` on the containing `sp-menu` or the `selects` property on the parent `sp-menu` or `sp-menu-group`. If the `selected` state is indicated on a `menuitem`  without `selects="multiple"`, `selects="single"` or `selects="inherit"` on the parent `sp-menu` or `sp-menu-group`, we should maybe change the `role` for the `sp-menu-item` to `menuitemcheckbox`. This gets messy though, because then the question becomes, what then is the `role` for the unselected state, and how do we indicate that an unselected item is selectable?

Reply:
@majornista To immediately solve the issue, can I make sure that the menu as display in this tray has the `selects` property? Then we can hook into the existing `aria-checked` logic, I can remove the `aria-current` logic I added, and the screen reader announces the selected item properly (which I think was the underlying complaint in the ticket).

I could also add more documentation guidelines around ensuring the `selects` property is added to the parent `sp-menu`, and describe the accessibility errors/failures that happen regarding `sp-menu-item`s when `selects` is missing.

I'm going to bring this up with our team this morning as well to chat about a solution!

Reply:
I can make a new ticket! I have a local branch that should still have some of this previous work.